aces/Loris
# [Core] Generically handle uncaught exceptions (no more blank crashes)

## Brief summary of changes

Modifies the middleware to handle uncaught exceptions in a generic way. **Instead of LORIS crashing and displaying a blank page, it should now display a decorated error template.**

I've also updated the error templates to encourage users to reach out to the project administrator or to report an issue using the issue tracker module.

### Examples of error handling

##### Example of how this fixes #6034 (Uncaught \NotFound exception)

<img width="741" alt="Screen Shot 2020-02-17 at 16 23 20" src="https://user-images.githubusercontent.com/4022790/74687810-7e609780-51a3-11ea-921e-f06215284308.png">

---

##### Example of how this fixes #4368 (Uncaught Internal Server Error)

<img width="1141" alt="Screen Shot 2020-02-17 at 18 00 28" src="https://user-images.githubusercontent.com/4022790/74691132-976f4580-51af-11ea-99f0-a8244bc480ee.png">

---

#### Testing instructions (if applicable)

1. Test either of the linked issues and make sure that the errors no longer cause crashes.
2. Go to a module that doesn't exist and confirm that LORIS doesn't crash.

#### Related

Fixes #4368
